### DISCUSSION TITLE
Users and radio stations missing after upgrade

### DISCUSSION THREAD

**Discussion by Alexyoe** [navidrome/navidrome#2239] [Q&A]
### Description

When upgrading from 0.49.1 to 0.49.3 all users including admin were removed as well as favorites and radio stations.

### Expected Behavior

After upgrade all users and radio stations remain as they were before upgrading.

### Steps to reproduce

1. Replace version 0.49.1 with version 0.49.3
2. Navigate to Navidrome via Chrome
3. Prompted to create a new admin user

### Platform information

 - Navidrome version: 0.49.3
 - Browser and version: Google Chrome Version 110.0.5481.178
 - Operating System: Windows 10 binary with Shawl

**Comment by deluan** [comment]
Looks like that somehow the database was lost when you replaced the Navidrome binary. This is not a bug in Navidrome, but maybe we need some more clarity in the documentation regarding updates?

**Reply by Alexyoe** [reply]
Is there a way to update the binary other than replacing it?

It looks like originally the database was being stored in `\System32` but after replacing the binary it's now being stored in `\SysWOW64`. I tried backing up the new DB and replacing it with the old one but that did not work. Not a huge issue as I had only added a handful of radio stations.

**Reply by deluan** [reply]
> Is there a way to update the binary other than replacing it?

Yes, that's how you do it, but it should not change the place where the database is stored. Maybe the Shawl configuration was changed as well?

>  tried backing up the new DB and replacing it with the old one but that did not work.

This should work. Stop Navidrome, replace the DB with the old one and restart Navidrome. You should see in the logs it applying any pending migrations and upgrading the DB.
